src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsChannelMapping.java

```diff
@@ -36,7 +36,8 @@ public static List<HearingChannel> getHearingChannels(@Valid SscsCaseData caseDa
     }
 
     public static HearingChannel getHearingChannel(@Valid SscsCaseData caseData) {
-        if (HearingsDetailsMapping.isPoOfficerAttending(caseData)) {
+        if (HearingsDetailsMapping.isPoOfficerAttending(caseData)
+            && isNull(OverridesMapping.getOverrideFields(caseData).getAppellantHearingChannel())) {
```

Comment:
If the override is there, dont you need to return that ?

Reply:
I think that it is returning Hearing channel according to situation? I mean on the 58-63 lines it is constructing everything from OverrideFields, or Im wrong?

Reply:
The override is for the appellant, not the case, so you don't need this.

Reply:
yes it is removed